packages/driver/src/cy/listeners.js

```diff
@@ -21,9 +21,16 @@ const removeAllListeners = () => {
 }
 
 const addListener = (win, event, cb) => {
-  events.push([win, event, cb])
+  const id = Math.random()
 
-  win.addEventListener(event, cb)
+  win.__cypressEventMapRPC = win.__cypressEventMapRPC || {}
+  win.__cypressEventMapRPC[id] = cb
```

Comment:
Is this really RPC?

> In distributed computing, a remote procedure call is when a computer program causes a procedure to execute in a different address space, which is coded as if it were a normal procedure call, without the programmer explicitly coding the details for the remote interaction.

`win.__cypressListenerMap`, `win.__cypressListenerCbs`, `win.__cypressEventCbMap`, `win.__cypressEventCbs` i feel like are more apt names

Reply:
Yeah, good point - I think I named it this was b/c we noted how this was similar to how other RPC implementations work in cross window communication, but in this case it's not.